src/s3/activation.ts

```diff
@@ -47,8 +47,15 @@ export async function activate(ctx: ExtContext): Promise<void> {
         Commands.register('aws.s3.presignedURL', async (node: S3FileNode) => {
             await presignedURLCommand(node)
         }),
-        Commands.register('aws.s3.downloadFileAs', async (node: S3FileNode) => {
-            await downloadFileAsCommand(node)
+        Commands.register('aws.s3.downloadFileAs', async (node: S3FileNode, allNodes?: S3FileNode[]) => {
+            if (allNodes) {
```

Comment:
is the separate `allNodes` parameter needed, or could `node` change to `nodes` and we check `size > 1` ?